tests/pc/test_lag_member_forwarding.py

```diff
@@ -0,0 +1,175 @@
+    exp_packet.set_do_not_care_scapy(scapy.Ether, "src")
+
+    exp_packet.set_do_not_care_scapy(scapy.IP, "version")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "ihl")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "tos")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "len")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "flags")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "id")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "frag")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "ttl")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "chksum")
+    exp_packet.set_do_not_care_scapy(scapy.IP, "options")
+
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "seq")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "ack")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "reserved")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "dataofs")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "window")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "chksum")
+    exp_packet.set_do_not_care_scapy(scapy.TCP, "urgptr")
+
+    exp_packet.set_ignore_extra_bytes()
+    return pkt, exp_packet
+
+
+def test_lag_member_forwarding_packets(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo, ptfadapter):
+    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
+    mg_facts = duthost.get_extended_minigraph_facts(tbinfo)
+    lag_facts = duthost.lag_facts(host=duthost.hostname)['ansible_facts']['lag_facts']
+    if not len(lag_facts['lags'].keys()):
+        pytest.skip("No Lag found in this topology")
+    portchannel_name = list(lag_facts['lags'].keys())[0]
+    portchannel_dest_name = None
+    recv_port = []
+    if len(lag_facts['lags'].keys()) > 1:
+        portchannel_dest_name = list(lag_facts['lags'].keys())[1]
+        portchannel_dest_members = list(lag_facts['lags'][portchannel_dest_name]['po_stats']['ports'].keys())
+        assert len(portchannel_dest_members) > 0
+        for member in portchannel_dest_members:
+            recv_port.append(mg_facts['minigraph_ptf_indices'][member])
+
+    portchannel_members = list(lag_facts['lags'][portchannel_name]['po_stats']['ports'].keys())
+    assert len(portchannel_members) > 0
+    asic_name = lag_facts['names'][portchannel_name]
+    dest_asic_name = lag_facts['names'][portchannel_dest_name]
+    asic_idx = duthost.get_asic_id_from_namespace(asic_name)
+    asichost = duthost.asic_instance_from_namespace(asic_name)
+    dest_asichost = duthost.asic_instance_from_namespace(dest_asic_name)
+
+    config_facts = asichost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
+    dest_config_facts = dest_asichost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
+    send_port = mg_facts['minigraph_ptf_indices'][portchannel_members[0]]
+    holdtime = 0
+
+    peer_device_ip_set = set()
+    peer_device_dest_ip = None
+
+    for peer_device_ip, peer_device_bgp_data in config_facts['BGP_NEIGHBOR'].items():
```

Comment:
please add some comments on what the following section does..

Reply:
@prabhataravind : added comments